test/TS/MovingAverage.lf

```diff
@@ -3,8 +3,7 @@
 // four inputs and produces that as output.  The source is a counting
 // sequence.
 target TypeScript {
-    timeout: 1 sec,
-    fast: true
```

Comment:
What's the reason for this change?

Reply:
This is probably a mistaken commit. I don't remember why I changed this, and the C test on which this is based has `fast: true`, so I don't know what's going on. I'll see if anything goes wrong when I change it back.